frontend/src/Map/MapTypeButton.js

```diff
@@ -1,23 +1,56 @@
-import React from 'react';
-import { LayersControl } from 'react-leaflet';
-import { TileLayer } from 'react-leaflet';
+import React, { useState, useEffect } from 'react';
+import { LayersControl, TileLayer } from 'react-leaflet';
+import HeatLayer from './HeatLayer';
+
+export default function MapTypeButton({ stations }) {
+  const [stationList, setStationList] = useState(null);
+
+  useEffect(() => {
```

Comment:
Btw, ez csak a szűrés miatt kell, én ezt egy filter-rel oldanám meg és nem itt, hanem a Map.js-ben.

Reply:
Ahogy egyébként ugyanez a szűrés megvan a Markers-ben is :D Lehet az a mondás, hogy a MapTypeButton és a Markers is csak olyan tömböt vár, amiben a longitude és a latitude nem null és ezt a szűrést elvégzi a Map.js nekik.

Reply:
Javítva , működik a heatmap de változott a színe (???)